### DISCUSSION TITLE
I lost my access to Copilot

### DISCUSSION THREAD

**Discussion by dent0n7887** [community/community#63119] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Today i have problems with copilot in PyCharm
https://disk.yandex.ru/i/ANZ-iQug6D4t4w

Error: You don't have access to GitHub Copilot. Sign up by visiting https://github.com/settings/copilot.

My account active, payment is ok.

**Comment by Akash1134** [comment]
Hi @dent0n7887 ,

Please check your PyCharm settings to ensure that GitHub Copilot is enabled and configured correctly.

If that does not help, there might be issues with your account. Please [reach out to our Support team](https://support.github.com/) to get it sorted.

Thanks!
